src/AboutUs.css

```diff
@@ -0,0 +1,77 @@
+.aboutUs{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    width: 80%;
+    margin-top: 110px;
+    margin-left: 10%;
+    margin-right: 10%;
+    perspective: 600px;
+    -moz-perspective: 600px
+}
+.myCard{
+   
+    padding-bottom: 5%;
+    perspective: 35rem;
+    width: 30%;
+    position: relative;
+    height: 30rem;
+    border-radius: 25px;
+
+    
+   
+    
+    /* box-shadow: 0 1.5rem 4rem rgba(0, 0, 0, .25); */
```

Comment:
Can we remove the extra lines with no code please and the commented out code that is not being used.